src/templates/category/ProductCard.js

```diff
@@ -31,7 +31,7 @@ class ProductCard extends Component {
       <div className="col-lg-4 col-md-6 col-sm-6 col-xs-12">
         <div className="demo-product-card">
           <Link to={`/product/${this.props.productId}`} style={{margin: '0'}}>
-            { this.props.children ||  <img src={this.props.images[0] ? this.props.images[0].originalSrc : require('../assets/images/default.jpeg')} alt={this.props.productName} />}
+            { this.props.children ||  <img src={this.props.images[0] ? this.props.images[0].originalSrc : require('../../assets/images/default.jpeg')} alt={this.props.productName} />}
```

Comment:
`<img src={this.props.images[0] ? this.props.images[0].originalSrc : require('../../assets/images/default.jpeg')` add a method and move this logic there and finally return only what to render.